test/other/test_emit_tsd.d.ts

```diff
@@ -0,0 +1,7 @@
+interface WasmModule {
```

Comment:
How about adding a comment to the of this file?

Something like `// TypeScript bindings for emscripten-generate code.  Automatically generated at compile time.`?